sdk/communication/Azure.Communication.MediaComposition/src/Models/ParticipantInput.cs

```diff
@@ -9,5 +10,33 @@ namespace Azure.Communication.MediaComposition.Models
     [CodeGenModel("Participant")]
     public partial class ParticipantInput
     {
+        [CodeGenMember("Id")]
+        internal CommunicationIdentifierModel ServiceId { get; set; }
+
+        /// <summary> The CommunicationIdentifier that identifies the participant.</summary>
+        public CommunicationIdentifier Id => CommunicationIdentifierSerializer.Deserialize(ServiceId);
```

Comment:
Keeping the CommunicationIdentifierModel type as the source of truth was the easiest fix, but it adds a small runtime cost whenever the property gets accessed. I guess I can make this a private setter and set it in the constructor.